tests/posts_tests/test_ratings.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from posts.models import Resume
+from pytest_django.asserts import assertTemplateUsed
+from conftest import RATER_USERNAME, RATER_PASSWORD, USERNAME, PASSWORD
+
+
+@pytest.mark.django_db()
+class TestRatingViews:
+    def test_get_rate_page_for_valid_resume(self, client, persist_resume, persist_rater):
+        resume = persist_resume
+        client.login(username=RATER_USERNAME, password=RATER_PASSWORD)
+        response = client.get(f'/post/{resume.pk}/rate/')
+        assert response.status_code == 200
+        assertTemplateUsed(response, 'posts/rating_form.html')
+
+    @pytest.mark.parametrize('invalid_resume', ['-1', '', 'resume'])
+    def test_get_rate_page_for_invalid_resume_path(self, client, invalid_resume, persist_rater):
+        client.login(username=RATER_USERNAME, password=RATER_PASSWORD)
+        response = client.get(f'/post/{invalid_resume}/rate/')
+        assert response.status_code == 404
+
+    def test_get_rate_page_for_not_exist_resume(self, client):
+        last_resume = Resume.objects.latest('post_id')
+        client.login(username=RATER_USERNAME, password=RATER_PASSWORD)
+        with pytest.raises(Resume.DoesNotExist):
+            client.get(f'/post/{last_resume.post_id+1}/rate/')
+
+    def test_rate_user_created_resume(self, client, persist_resume):
+        # Note: USERNAME, PASSWORD are the username and passoword of the resume creator
```

Comment:
maybe change it here to doc string?

Reply:
I thought about it, but this comment is more of an explanation of the `USERNAME` and `PASSWORD` variables, not an explanation of the function 🙂